src/pages/CartPage/CartPage.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './CartPage.css';
+import {Nav} from './../../components/Nav/Nav';
+import { VerticalCard } from '../../components/Card/VerticalCard';
```

Comment:
You can use an `index.js` file in the components folder and add all your components exports in that. By this you can import all your components from a single file.

Reply:
i will implement that.